.github/workflows/verify-app.yml

```diff
@@ -0,0 +1,62 @@
+name: 'Capture Growth Chart: verify app'
+
+on:
+  push:
+    branches:
+
+jobs:
+  install:
```

Comment:
More specific names for the jobs:

- `install` -->   `install_dependencies`
- `lint` -->   `lint_code`
- `typescript` -->   `typescript_check`